src/components/Search/SearchView/SearchView.css

```diff
@@ -0,0 +1,11 @@
+.MuiGrid-grid-xs-11 {
+  z-index: 2;
+}
+
+.MuiGrid-grid-xs-12 {
+  position: absolute;
+  z-index: 1;
+
+  top: 200px;
+  width: 70%;
```

Comment:
change this width to 90% to better center the table. Everything else looks good :ok_hand: